File: cpp/matrix.cpp
#include<iostream>
using namespace std;

int main()
{
	int matrix[3][4]={
		{1,2,3,4},
		{5,6,7,8},
		{9,1,2,5},
	};
	int rows=sizeof(matrix)/sizeof(matrix[0]);
	int cols=sizeof(matrix)/(rows*sizeof(matrix[0][0]));   
	for (int i = 0; i < rows; i++)
	{
		for (int j = 0; j < cols; j++)
		{
			cout<<matrix[i][j]<<" ";
		}
			cout<<"\n";
	}
    return 0;
}

File: python/matrix.py
matrix=[
[1,2,3,4],
[5,6,7,8],
[9,10,11,12],
]
rows=len(matrix)
cols=len(matrix[0])
for i in range(0,rows):
	for j in range(0,cols):
		print(f"{matrix[i][j]}", end=" ")
	print("\n")